content/docs/04.5-devx/05-app-profile/02-container-deployment.md

```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
 
 <br />
 
-1. Log in to the Palette Dev Engine console
+# Prerequisite
```

Comment:
We also need an app profile so we should list it here and link to the how to create an app profile page.